### PR TITLE
Ci integration

### PR DESCRIPTION
**Pull request by nastasi-oq**
With this PR we add CI integration and use of selenium tests imported from external platform plugin.
A big change is the renaming of `openquakeplatform_standalone` package in `openquakeplatform`
to reduce differences with `oq-platform` package.
Test are green here: https://ci.openquake.org/job/z-devel_oq-platform-standalone/6/

### CODE REVIEW

File: openquakeplatform/test/__init__.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import time
+import sys
+import nose
```
Comment (daniviga) [inline openquakeplatform/test/__init__.py]: These imports seem to be unused

File: openquakeplatform/test/oqplatform.py [hunk 2]
```diff
@@ -0,0 +1,425 @@
...
+        from selenium.common.exceptions import StaleElementReferenceException
+
+        def link_has_gone_stale():
+            try:
+                # poll the link with an arbitrary call
+                element.find_elements_by_id('doesnt-matter')
+                return False
+            except StaleElementReferenceException:
+                if (self.driver.current_url == url
+                        or self.driver.current_url == (self.basepath + url)):
```
Comment (daniviga) [inline openquakeplatform/test/oqplatform.py:320]: The 'or' should be on the line above to be pep8 compliant

File: openquakeplatform/test/satellites_test.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    return mod
+
+def get_checks(pkgname):
+    cls = import_by_name(pkgname)
+    for objname in dir(cls):
+         obj = getattr(cls, objname)
+         if isclass(obj) and issubclass(obj, unittest.TestCase):
+             newname = "%s__%s" % (pkgname.replace('.', '__'), objname)
+             globals()[newname] = obj
+             obj.__name__ = newname
```
Comment (daniviga) [inline openquakeplatform/test/satellites_test.py]: Indentation from 13 to 17 looks a bit strange. Is it multiple of 4?

-> flake8 says no, please fix it (and add a new line before each def, they should be separated by two blank lines)

File: openquakeplatform/test/utils.py [hunk 4]
```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
+import time
+import sys
```
Comment (daniviga) [inline openquakeplatform/test/utils.py]: Unused import `sys`

File: verifier.sh [hunk 5]
```diff
@@ -0,0 +1,686 @@
...
+
+#
+#  sig_hand - manages cleanup if the build is aborted
+#
+sig_hand () {
+    trap ERR
+    echo "signal trapped"
+    if [ "$lxc_name" != "" ]; then
+        set +e
+        ssh -t  $lxc_ip "cd ~/$GEM_GIT_PACKAGE; . platform-env/bin/activate"
```
Comment (daniviga) [inline verifier.sh]: This activates a virtual environment in a session that dies immediately.  It looks useless.

File: verifier.sh [hunk 6]
```diff
@@ -0,0 +1,686 @@
...
+#                     - installs oq-engine sources on lxc
+#                     - set up postgres
+#                     - upgrade db
+#                     - runs celeryd
+#                     - runs tests
+#                     - runs coverage
+#                     - collects all tests output files from lxc
+#
+#      <branch_id>    name of the tested branch
+#      <lxc_ip>       the IP address of lxc instance
```
Comment (daniviga) [inline verifier.sh:359]: We should fix this comment (or just remove the detailed workflow)

File: verifier.sh [hunk 7]
```diff
@@ -0,0 +1,686 @@
+#!/bin/bash
+#
+# packager.sh  Copyright (c) 2015, GEM Foundation.
```
Comment (daniviga) [inline verifier.sh]: 2016

File: verifier.sh [hunk 8]
```diff
@@ -0,0 +1,686 @@
...
+
+    ssh -t  $lxc_ip "rm -f ssh.log"
+
+    ssh -t  $lxc_ip "sudo apt-get update"
+    ssh -t  $lxc_ip "sudo apt-get -y upgrade"
+
+    ssh -t  $lxc_ip "sudo apt-get install -y build-essential python-dev python-imaging python-virtualenv git libxml2 libxml2-dev libxslt1-dev libxslt1.1 libblas-dev liblapack-dev curl wget xmlstarlet imagemagick gfortran python-nose libgeos-dev python-software-properties"
+    ssh -t  $lxc_ip "sudo add-apt-repository -y ppa:openquake-automatic-team/latest-master"
+    ssh -t  $lxc_ip "sudo apt-get update"
+    ssh -t  $lxc_ip "sudo apt-get install -y python-decorator python-h5py python-psutil python-concurrent.futures python-oq-hazardlib python-oq-engine"
```
Comment (daniviga) [inline verifier.sh]:
``` bash
ssh -t  $lxc_ip "sudo apt-get install -y python-oq-hazardlib python-oq-engine"
```

is enough and then we don't need to update dependencies across several scripts if something changes in the deps

File: verifier.sh [hunk 9]
```diff
@@ -0,0 +1,686 @@
...
+"
+    echo "_prodtest_innervm_run: exit"
+
+    return 0
+}
+
+
+
+#
+#  devtest_run <branch_id> - main function of source test
```
Comment (daniviga) [inline verifier.sh]: Fix the comment with the right name

File: openquakeplatform/test/oqplatform.py (hunk 2 above)
Comment (nastasi-oq) [reply openquakeplatform/test/oqplatform.py:320]: Pep8 tool is not following latest pep8 rules, we already do it:
  https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator

File: openquakeplatform/test/satellites_test.py (hunk 3 above)
Comment (nastasi-oq) [reply openquakeplatform/test/satellites_test.py]: Right, fixed.

File: verifier.sh (hunk 5 above)
Comment (nastasi-oq) [reply verifier.sh]: +1, removed.

File: openquakeplatform/test/oqplatform.py (hunk 2 above)
Comment (daniviga) [reply openquakeplatform/test/oqplatform.py:320]: Good to know! Thanks

File: verifier.sh (hunk 6 above)
Comment (nastasi-oq) [reply verifier.sh:359]: Fixed.

File: verifier.sh (hunk 8 above)
Comment (nastasi-oq) [reply verifier.sh]: Ok.

**Comment by daniviga** [comment]
LGTM

**Pull request merged**